apollo-federation/src/sources/connect/models/validation/mod.rs

```diff
@@ -42,16 +42,18 @@
 - Missing $this
 
 */
+mod entities;
```

Comment:
This probably won't have _all_ entity checks in it (e.g., `@key` stuff), so maybe this should be `entity` to refer to the arg?